.github/workflows/sandbox.yml

```
@@ -98,6 +34,7 @@ jobs:
             ghcr.io/${{ github.repository_owner }}/flyte-sandbox
           tags: |
             dind
+            ${{ steps.set_version.outputs.flyte_version }}
```

Should we also add this now that we will only publish the dind images?

```suggestion
            ${{ steps.set_version.outputs.flyte_version }}
            type=sha,format=long
```

---

we need prefix in sha, it's old structure

---

@EngHabu There is one issue https://github.com/flyteorg/flytectl/blob/master/pkg/util/githubutil/githubutil.go#L224